packages/fxa-settings/src/models/Account.ts

```diff
@@ -400,6 +400,12 @@ export class Account implements AccountData {
     );
   }
 
+  async getAccountStatusByEmail(email: string): Promise<any> {
```

Comment:
Originally in Backbone, we would check to make sure the account associated with the email existed. While I opted to simplify the page by omitting this per discussion with @vbudhram, we will need this method.

Reply:
nit: Does this need to be `any` or can it be more specific?

Reply:
Hmmmm I think it can be a boolean actually, checking!

Reply:
Fixed!